### PR TITLE
add identifiers to authors

### PR DESCRIPTION
**Pull request by jum-s**
the current state of this pr is intended to be a base of discussion to start integrating some identifier to authors layout
with the idea to improve tasks suggestions (as described in #501)
maybe we could wrap the identifiers to only display it on a user's click, maybe we could only display it in a taks context, maybe its fine like this now, i dont have a strong opinion on any of those suggestions

### CODE REVIEW

File: app/lib/handlebars_helpers/identifiers.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+export default {
+  'wdt:P213': id => { return `https://isni.oclc.org/xslt/DB=1.2//CMD?ACT=SRCH&IKT=8006&TRM=ISN%3A${id}` },
```
Comment (maxlath) [inline app/lib/handlebars_helpers/identifiers.js:2]: syntactic preference would be to use the arrow function implict return for the beauty of that simpler one-liner
```suggestion
  'wdt:P213': id => `https://isni.oclc.org/xslt/DB=1.2//CMD?ACT=SRCH&IKT=8006&TRM=ISN%3A${id}`,
```

**Comment by maxlath** [comment]
I'm all in favour of making those ids and URLs available to people who would appreciate it, but there are a lot of external ids and it will end up being extremely cluttered for most uses: I was thinking we could make this choice possible in user preferences; the user could check/uncheck all the properties to use to fit there interest. I would, for instance, probably keep the BNF activated, but disable instragram, facebook, and most libraries and databases I don't really use and could still easily access via Wikidata, without paying the prices of a page full of ids

File: app/lib/handlebars_helpers/identifiers.js (hunk 1 above)
Comment (jum-s) [reply app/lib/handlebars_helpers/identifiers.js:2]: yeah i know, but somehow webpack doesnt like it and fails to compile...

Comment (maxlath) [reply app/lib/handlebars_helpers/identifiers.js:2]: what's webpack error message?

**Comment by jum-s** [comment]
Would you consider a entities tab in the settings. Somewhere around : https://github.com/inventaire/inventaire-client/blob/master/app/modules/settings/views/settings_layout.js#L40

**Comment by maxlath** [comment]
That could be on the display settings page, in a subsection named "entities" or some category name that could makes more sense to new comers

**Comment by jum-s** [comment]
since display setting page is not yet on master, i would wait for #205 to be merged

File: app/lib/handlebars_helpers/identifiers.js (hunk 1 above)
Comment (jum-s) [reply app/lib/handlebars_helpers/identifiers.js]: done 884c4b4

**Comment by jum-s** [comment]
done with #339

**Pull request closed without merging**